CONTRIBUTING.md

```diff
@@ -2,31 +2,112 @@
 
 First, thanks for taking the time to contribute to the project! Your support is highly appreciated - kudos!
 
-The following is a set of guidelines for contributing to KubeCF and its modules. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+The following is a set of guidelines for contributing to KubeCF and its modules. These are mostly guidelines, not rules.
+Use your best judgment and feel free to propose changes to this document in a pull request.
 
 ## How to contribute
 
-Before kicking the ball, please **do NOT report security vulnerabilities in public issues**! Instead, report with the CloudFoundry Foundation team first <security@cloudfoundry.org> and give us some time to fix it in a fashion time matter before disclosing it to the public. For more information check the CloudFoundry Security [page](https://www.cloudfoundry.org/security/).
+Before kicking the ball, please **do NOT report security vulnerabilities in public issues**! Instead, please report with the
+CloudFoundry Foundation team first <security@cloudfoundry.org> and give us some time to fix it in a fashion time matter
+before disclosing it to the public. 
+For more information, check the CloudFoundry Security [page](https://www.cloudfoundry.org/security/).
 
-When contributing to this repository, please first discuss the changes through an existent Github issue with the main contributers.
+When contributing to this repository, please first discuss the changes through an existing Github issue with the main
+contributors.
 
 ## Issues tracker
 
-We're commited to have short and accurate templates for different type issues, in order to gather the required information to start a conversation.
+We're commited to have short and accurate templates for different type issues in order to gather the required
+information to start a conversation.
 
 Once again, feel free to contribute to improve them by opening a pull request.
 
 ### How to report a bug
 
-Start by searching [bugs][1] for some similiar issues and/or extra information. If your search doesn't bring you any help then open an issue by selecting the issue type "Bug" and fill the template accurately as possible.
+Start by searching [bugs][1] for some similiar issues and/or extra information. If your search doesn't bring you any
+help, then open an issue by selecting the issue type "Bug" and fill the template accurately as possible.
 
 ### How to suggest a feature/enhancement
 
-If you find yourself wishing for a feature/enhancement that does not exist yet in KubeCF start by run a quick [search][2] - you may not be alone! If you ain't any luck, then go ahead and open a new issue by selecting the "Feature" issue type and answer some needed questions.
+If you find yourself wishing for a feature/enhancement that does not exist yet in KubeCF, start by run a
```

Comment:
If you would like to request for a feature that does not yet exist in KubeCF...

Reply:
> If you would like to request  a feature that does not yet exist in KubeCF...
(removed `for`)